serialization_test.go

```diff
@@ -239,11 +239,19 @@ func TestJSONError(t *testing.T) {
 
 func TestSend(t *testing.T) {
 	w := httptest.NewRecorder()
-	SendText(w, "Hello World")
+	SendText(w, nil, "Hello World")
 
 	require.Equal(t, "Hello World", w.Body.String())
```

Comment:
I would like to recommend you to test the content type here and in all other tests, such as : 
-  TestSendTextError
- TestSendYAML
- TestSendJSON

Please note  TestSendYAMLError and TestSendHTMLError already test the content type